stripe/src/main/java/com/stripe/android/utils/ClassUtils.java

```diff
@@ -15,7 +15,7 @@ private ClassUtils() {}
      * @param clazz the class to search in
      * @param whitelist the whitelist of field names
      * @param obj the target object whose field we are accessing
-     * @return  the value of the found field, if exists, on the specified {@param obj}, or null
+     * @return  the value of the found field, if exists, on the specified {@code obj}, or null
```

Comment:
revert

Reply:
I don't care either way but this was raising checkstyle issues